src/dashboard/src/components/api/views.py

```diff
@@ -157,23 +182,14 @@ def status(request, unit_uuid, unit_type):
     return helpers.json_response(response)
 
 
+@_api_endpoint(expected_methods=['GET'])
 def waiting_for_user_input(request):
```

Comment:
Despite the url (`/api/ingest/waiting`) this returns all units, SIP or Transfer, waiting for user input. Should we add a transfer vs ingest endpoint and limit results?